docs/router/metrics-and-monitoring.mdx

```diff
@@ -214,8 +214,8 @@ telemetry:
 * `router.engine.messages.sent`: The number of total messages for subscriptions sent over from the subgraph to the router.
 
 
-### EDFS Streams Metrics
-We expose metrics for EDFS streams, these statistics are collected at the level when the message is sent to the messaging backend or directly received from the messaging backend.
+### Cosmo Streams Metrics
+We expose metrics for [Cosmo Streams](/router/cosmo-streams). These statistics are collected at the level when the message is sent to the messaging backend or directly received from the messaging backend.
```

Comment:
I think `collected` is not the right term, because metrics are either pulled (Prometheus) or pushed (OTEL).
We just make sure to expose those metrics for the corresponding provider to collect those

```suggestion
We expose metrics for [Cosmo Streams](/router/cosmo-streams). New data points are emitted when new messages are either sent or received from the messaging backend
```